images/win/toolsets/toolset-2022.json

```diff
@@ -242,14 +242,16 @@
             "Microsoft.VisualStudio.Component.VC.14.29.16.11.ATL.Spectre",
             "Microsoft.VisualStudio.Component.VC.14.29.16.11.MFC",
             "Microsoft.VisualStudio.Component.VC.14.29.16.11.MFC.Spectre",
+            "Microsoft.VisualStudio.Component.VC.14.32.17.2.ARM",
+            "Microsoft.VisualStudio.Component.VC.14.32.17.2.ARM.Spectre",
+            "Microsoft.VisualStudio.Component.VC.14.32.17.2.ARM64",
+            "Microsoft.VisualStudio.Component.VC.14.32.17.2.ARM64.Spectre",
             "Microsoft.VisualStudio.Component.VC.14.32.17.2.x86.x64",
             "Microsoft.VisualStudio.Component.VC.14.32.17.2.x86.x64.Spectre",
             "Microsoft.VisualStudio.Component.VC.14.32.17.2.ATL",
             "Microsoft.VisualStudio.Component.VC.14.32.17.2.ATL.Spectre",
             "Microsoft.VisualStudio.Component.VC.14.32.17.2.MFC",
             "Microsoft.VisualStudio.Component.VC.14.32.17.2.MFC.Spectre",
-            "Microsoft.VisualStudio.Component.VC.14.32.17.2.x86.x64",
```

Comment:
For ARM/ARM64 targeting, you'll probably also want to add:

            "Microsoft.VisualStudio.Component.VC.14.32.17.2.ATL.ARM",
            "Microsoft.VisualStudio.Component.VC.14.32.17.2.ATL.ARM.Spectre", 
            "Microsoft.VisualStudio.Component.VC.14.32.17.2.ATL.ARM64",
            "Microsoft.VisualStudio.Component.VC.14.32.17.2.ATL.ARM64.Spectre",               
            "Microsoft.VisualStudio.Component.VC.14.32.17.2.ATL.ARM64EC",
            "Microsoft.VisualStudio.Component.VC.14.32.17.2.ATL.ARM64EC.Spectre",               
            "Microsoft.VisualStudio.Component.VC.14.32.17.2.MFC.ARM",
            "Microsoft.VisualStudio.Component.VC.14.32.17.2.MFC.ARM.Spectre",
            "Microsoft.VisualStudio.Component.VC.14.32.17.2.MFC.ARM64",
            "Microsoft.VisualStudio.Component.VC.14.32.17.2.MFC.ARM64.Spectre",               
            "Microsoft.VisualStudio.Component.VC.14.32.17.2.MFC.ARM64EC",
            "Microsoft.VisualStudio.Component.VC.14.32.17.2.MFC.ARM64EC.Spectre",

Reply:
Updated. Thank you.

Reply:
There are no Microsoft.VisualStudio.Component.VC.14.32.17.2.X.ARM64EC components for VS 2022 - https://docs.microsoft.com/en-us/visualstudio/install/workload-component-id-vs-enterprise?view=vs-2022